.docker/entrypoint/sh/entrypoint.sh

```diff
@@ -1,15 +1,42 @@
-#!/bin/bash
-set -o errexit # Immediately abort if any command fails
+#!/usr/bin/env bash
+set -o errexit # Abort if any command fails
 
-[ "${DEBUG:-false}" = "true" ] && set -x
+parse_args() {
+    if [[ "$@" =~ \-\-env[\=\ ]+([^\ ]+) ]]; then
+        env="${BASH_REMATCH[1]}"
+        [ ${DEBUG:-false} ] && echo "Detected --env='$env'."
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)